docs/Development/ReleaseNotes.md

```diff
@@ -54,55 +54,59 @@ This is a minor update from VIC 5.0.1. The VIC 5.1.0 includes new features, such
     - Fixed bug that prevented using the correct local domain grid cells in `cesm_put_data.c`
     - Changed reference temperature units from Celsius to Kelvin in `cesm_put_data.c`
 
-   1. [GH#695](https://github.com/UW-Hydro/VIC/pull/695)
+   2. [GH#695](https://github.com/UW-Hydro/VIC/pull/695)
```

Comment:
As @jhamman says. Leave the `1` for all the numbered entries. The `1` just indicates that markdown should format this as an enumerated list and will take care of the actual numbering. It is much easier to leave that to markdown in case the list ever gets updated.